### PR TITLE
feat: Set DefaultPairs with defaultable pairs

### PR DESCRIPTION
**Pull request by JinnyYi**
ref: https://github.com/beyondstorage/go-storage/issues/704

### CODE REVIEW

File: cmd/definitions/tmpl/service.tmpl [hunk 1]
```diff
@@ -245,6 +248,33 @@ func parsePair{{ $pn }}{{ $fnk }}(opts []Pair) (pair{{ $pn }}{{ $fnk }}, error)
     }
     {{- end }}
 
+    // Default pairs
+    {{- range $_, $pair := $dp }}
+    if result.has{{ $pair.Name | toPascal }} {
+        result.HasDefault{{ $pn }}Pairs = true
+        {{- range $_, $op := $fu }}
+            {{- range $_, $opPair := $op.Required }}
+                {{- if eq $pair.Name (print "default_" $opPair.Name) }}
```
Comment (Xuanwo) [inline cmd/definitions/tmpl/service.tmpl:257]: Why? I thought we have `default_xxx` pairs already.

Comment (JinnyYi) [reply cmd/definitions/tmpl/service.tmpl:257]: I'm not so sure about your query.
The intention here is to find operations that contains the default pair, and the pairs for operations don't contain `default_` prefix.

Comment (Xuanwo) [reply cmd/definitions/tmpl/service.tmpl:257]: I'm a bit confused, let's discuss in https://matrix.to/#/#beyondstorage@gsp-700:matrix.org

File: cmd/definitions/type.go [hunk 2]
```diff
@@ -85,19 +85,34 @@ type Namespace struct {
...
-func (n *Namespace) Defaultable() []*Pair {
-	ps := make([]*Pair, 0, len(n.defaultable))
+// Defaultable returns defaultable pair and operations map.
+func (n *Namespace) Defaultable() map[*Pair][]string {
+	pf := make(map[*Pair][]string)
+
 	for _, v := range n.defaultable {
-		ps = append(ps, v)
+		var ops []string
+		for _, op := range n.Funcs {
```
Comment (Xuanwo) [inline cmd/definitions/type.go:94]: How about

```go
ps := make([]*Paris, 0)
ps = append(ps, op.Required...)
ps = append(ps, op.Optional...)

for _, v := range ps {
    ...
}
```

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**